release_notes/topics/product/7_4_final.adoc

```diff
@@ -100,9 +108,16 @@ standalone -Dkeycloak.profile.feature.account_api=enabled -Dkeycloak.profile.fea
 
 When you go to Manage Account, you will see the new Account Console.  Here is a sample screen:
 
-image:{project_images}/device-activity.png[]
+.Device activity
+image:{project_images}/device-activity.png[Device activity]
+
+This new console is based on React and PatternFly 4.  It allows you to use PatternFly CSS variables for easy styling.  It also allows you to remove pages and add your own pages.  Full documentation will be provided at a future release.
 
-This new console is based on link:https://reactjs.org/[React] and link:https://www.patternfly.org/v4/[PatternFly 4].  It allows you to use link:https://www.patternfly.org/v4/developer-resources/global-css-variables/#global-variables[PatternFly CSS variables] for easy styling.  It also allows you to remove pages and add your own pages.  Full documentation will be provided at a future release.
+.Additional Resources
```

Comment:
The casing of the title "Additional Resources" is different to others "Additional resources". I am not sure about which one is correct as per the style guide.

Reply:
Thanks, @nehachopra27 It should be lower case always